src/resources/authentication/checkAuthentication.js

```diff
@@ -29,6 +30,14 @@ const checkAuthentication = (req, res, next) => {
     return next();
   }
 
+  if (req.path === REVIEW_PATH && req.method === 'GET') {
+    return next();
+  }
+
+  if (req.path === REVIEW_PATH && req.method === 'PUT') {
```

Comment:
only authorized users should be able to post comments. with this condition you allow to leave comments for everyone

Reply:
button "submit new review" available only autorized user